app/src/main/java/framgia/com/ichat/screen/home/HomeActivity.java

```diff
@@ -1,15 +1,61 @@
 package framgia.com.ichat.screen.home;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.design.widget.TabLayout;
+import android.support.v4.view.ViewPager;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
 
 import framgia.com.ichat.R;
+import framgia.com.ichat.screen.base.BaseActivity;
 
-public class HomeActivity extends AppCompatActivity {
+public class HomeActivity extends BaseActivity implements android.support.v7.widget.SearchView.OnQueryTextListener {
+    private Toolbar mToolbar;
+    private android.support.v7.widget.SearchView mSearchView;
```

Comment:
import android.support.v7.widget.SearchView
'
```suggestion
    private SearchView mSearchView;
```